modules/simulation/src/main/resources/schemas/neurosciencegraph/simulation/morphology/v0.1.3.json

```diff
@@ -0,0 +1,87 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/simulation/morphology/v0.1.3/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.2"
+  ],
+  "shapes": [
+    {
+      "@id": "this:MorphologyShape",
+      "@type": "sh:NodeShape",
+      "label": "This shape is for...",
+      "targetClass": "nsg:Morphology",
+      "nodekind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.1/shapes/ModelInstanceShape"
```

Comment:
Update to {{base}}/schemas/neurosciencegraph/commons/modelinstance/v0.1.2

Reply:
fixed: https://github.com/INCF/neuroshapes/pull/119